microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -431,6 +458,135 @@ func (r *GMConnectorReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	return ctrl.Result{}, nil
 }
 
+// get the svc NamespaceName from the svcCfg and replace the ones in the original config map
+func preprocessUserConfigmap(ns string, hwType string) error {
```

Comment:
As we discussed,  `preprocessUserConfigmap` is a common function to handle the `configmap` from oneclick, so @KfreeZ will draw a new function, such as named `AAA`,  to handle the hardcode endpoint. And I will refactor the `getCustomConfig` to overwrite the `configmap` based on the service config provided by GMC CR yaml.